routes/sites.js

```diff
@@ -59,24 +59,28 @@ router.get('/', async function(req, res, next) {
     // Variables to track pagination of user's repos in case user has more than 100
     let pageCount = 1
     let hasNextPage = true;
-    const filePath = `https://api.github.com/user/repos?per_page=100&page=`;
+    const endpoint = `https://api.github.com/orgs/${ISOMER_GITHUB_ORG_NAME}/repos`;
 
     // Loop through all user repos
     while (hasNextPage) {
-      const resp = await axios.get(filePath + pageCount, {
+      const resp = await axios.get(endpoint, {
+        params: {
+          per_page: 100,
+          page: pageCount,
+          sort: "full_name",
+        },
         headers: {
           Authorization: `token ${access_token}`,
-          "Content-Type": "application/json"
+          Accept: "application/vnd.github.baptiste-preview+json",
```

Comment:
is `baptiste_preview` necessary? That media type is for template repositories.
![image](https://user-images.githubusercontent.com/7150533/70224892-fe708700-1788-11ea-8546-2b15d1c6b867.png)

Reply:
Not necessary, will remove in a separate PR

Reply:
resolved in #44